_scripts/check.sh

```diff
@@ -46,3 +46,15 @@ elif [ "$os" = "Linux" ]; then
       echo "sudo service docker restart"
     fi
 fi
+
+if [[ $(which gcloud) && $(gcloud --version) ]]; then
+  gcloud=y
+else
+  gcloud=n
+fi
+
+if [ "$gcloud" = "n" ]; then
+  echo "install gcloud to continue installation"
```

Comment:
you could also automate the installation for osx and / or darwin based on instructions in https://cloud.google.com/sdk/docs/quickstart-linux and https://cloud.google.com/sdk/docs/quickstart-macos

Reply:
I had considered that, but I don't have an OSX box to test it with, and their d/l links are pegged to versions which move fairly quickly so then we'd have to keep updating it.